pkg/recovery/permissiverecoverer.go

```diff
@@ -0,0 +1,37 @@
+package recovery
+
+import (
+	"log"
+	"os/exec"
+)
+
+const shell string = "rush"
```

Comment:
I would make this a public field of `PermissiveRecoverer` so it can be set by the caller. If it is set to something, run it, otherwise panic (and you can remove `Debug`). You could call this field `Command`, `RecoveryCmd` or similar name

Reply:
Donbe